pkg/services/ldap/ldap.go

```diff
@@ -140,7 +140,7 @@ func (server *Server) Dial() error {
 
 // Close closes the LDAP connection
 func (server *Server) Close() {
-	server.Connection.Close()
+	// server.Connection.Close()
```

Comment:
Should this be commented?

Reply:
Hm... Good catch, looks like I tried to fix problem that #18897 fixes. I have to revert this of course.